lib/rest_openshift.rb

```diff
@@ -110,42 +108,26 @@ def self.rollback_deploy(base_opts, opts)
 
         project_name = opts[:project_name]
 
-        populate("/namespaces/<project_name>/deploymentconfigrollbacks", base_opts, opts)
+        populate("/namespaces/<project_name>/deploymentconfigrollbacks", base_opts, opts, type="apps")
         return Http.request(**base_opts, method: "POST")
       end
 
-      def self.get_subresources_oapi(base_opts, opts)
-        populate("/namespaces/<project_name>/<resource_type>/<resource_name>/status", base_opts, opts)
-        return Http.request(**base_opts, method: "GET")
-      end
-
-      def self.post_role_oapi(base_opts, opts)
```

Comment:
@pruan-rht any replacement for this method?

Reply:
@stuartchuan, for 4.0, oapi has been removed, that's why I took out this method